File: httpsnippet/src/test/resources/output/node/request/query.js
var request = require("request");

var options = {"qs":{"foo":["bar","baz"],"baz":["abc"]},"method":"GET","url":"http://mockbin.com/har?key=value"};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: httpsnippet/src/test/resources/output/ruby/nethttp/query.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body
